### PR TITLE
Port RoboFile improvements from client project 

### PR DESCRIPTION
**Pull request by AronNovak**
#50

### CODE REVIEW

File: RoboFile.php [hunk 1]
```diff
@@ -339,10 +351,10 @@ public function phpcs() {
     $directories = [
       'modules/custom',
       'themes/custom',
-      'profiles/custom'
+      'profiles/custom',
     ];
 
-    $errorCode = null;
+    $errorCode = NULL;
```
Comment (amitaibu) [inline RoboFile.php:357]:
```suggestion
    $error_code = NULL;
```

File: RoboFile.php [hunk 2]
```diff
@@ -238,7 +243,7 @@ public function deployPantheon($branchName = 'master') {
       'travis-key',
     ];
 
-    $rsyncExcludeString = '--exclude=' . join(' --exclude=', $rsyncExclude);
+    $rsyncExcludeString = '--exclude=' . implode(' --exclude=', $rsyncExclude);
```
Comment (amitaibu) [inline RoboFile.php:246]: change to snake_case

**Comment by AronNovak** [comment]
@amitaibu That would require more work, I could not return back to that, introducing that is not obvious, as it's not part of Terminus itself. I guess it's better to keep Drupal Starter up-to-date early, as this race condition leads to build failures.

**Comment by amitaibu** [comment]
> I guess it's better to keep Drupal Starter up-to-date early, as this race condition leads to build failures.

Ok, sounds good.

**Comment by AronNovak** [comment]
@amitaibu There were lots of camelCase occurrences, I eliminated them all hopefully now.

**Pull request merged**